test/blackbox-tests/test-cases/pkg/pin-stanza/update-non-dune-local-pin.t

```diff
@@ -51,8 +54,11 @@ Update the message that gets printed while building foo:
   > EOF
 
 The change to the package is picked up:
-  $ dune build 2>&1 | grep -v make
+  $ dune build --debug-package-logs 2>&1 | grep -v make
   echo bbb
   bbb
   false
-  -> required by _build/_private/default/.pkg/foo/target/cookie
+  File "dune.lock/foo.pkg", line 4, characters 6-13:
+            ^^^^^^^
+  Error: Logs for package foo
```

Comment:
The loc doesn't seem right here (`^^^` points at nothing). Let's move that to a new issue.